decidim_app-design/app/views/public/evote-poll-landing.html.erb

```diff
@@ -9,12 +9,12 @@
   <div class="wrapper">
     <div class="row">
       <div class="columns">
-        <a href="#">
-          <%= icon "caret-left", class: "icon--small" %>&nbsp;Todas las votaciones
+        <a href="evote-poll-list">
```

Comment:
Just a little thing here, why the language of design's template has been changed?
I saw in next design's templates that same language has been kept..